src/main/java/samza/examples/azure/data/PageViewAvroRecord.java

```diff
@@ -0,0 +1,48 @@
+package samza.examples.azure.data;
+
+import java.io.Serializable;
+import org.apache.avro.AvroRuntimeException;
+import samza.examples.cookbook.data.PageView;
+
+public class PageViewAvroRecord extends org.apache.avro.specific.SpecificRecordBase
+    implements org.apache.avro.specific.SpecificRecord, Serializable {
+  public final org.apache.avro.Schema SCHEMA = org.apache.avro.Schema.parse(
+      "{\"type\":\"record\",\"name\":\"PageViewAvroRecord\",\"namespace\":\"org.apache.samza.examples.events\", \"fields\":[{\"name\": \"userId\", \"type\": \"string\"}, {\"name\": \"country\", \"type\": \"string\"}, {\"name\": \"pageId\", \"type\": \"string\"}]}");
+
+  private String userId;
+  private String country;
+  private String pageId;
+
+  public static PageViewAvroRecord buildPageViewRecord(PageView pageView) {
+    PageViewAvroRecord record = new PageViewAvroRecord();
+    record.put(0, pageView.userId);
```

Comment:
Would it be cleaner to do `record.userId = pageView.userId`? Same for the other fields below.